core/controllers/profile.py

```diff
@@ -365,18 +365,11 @@ def post(self):
                 raise self.InvalidInputException(e)
 
         if can_receive_email_updates is not None:
-            bulk_email_signup_message_should_be_shown = (
-                user_services.update_email_preferences(
-                    self.user_id, can_receive_email_updates,
-                    feconf.DEFAULT_EDITOR_ROLE_EMAIL_PREFERENCE,
-                    feconf.DEFAULT_FEEDBACK_MESSAGE_EMAIL_PREFERENCE,
-                    feconf.DEFAULT_SUBSCRIPTION_EMAIL_PREFERENCE))
-            if bulk_email_signup_message_should_be_shown:
```

Comment:
Why did this part change?

Reply:
Oh, this is related to the signup handler, where if a new user registers with the same email with which they already had an account at Oppia, that they deleted permanently earlier. This was removed accidentally, will add it back.